AxonDeepSeg/params.py

```diff
@@ -7,22 +7,32 @@
     'background': np.iinfo(np.uint8).min
 }
 
-# morphometrics column names
-column_names = np.array(
-    [],
-    dtype=[
-        ('x0 (px)', 'f4'),
-        ('y0 (px)', 'f4'),
-        ('gratio', 'f4'),
-        ('axon_area (um\u00b2)', 'f4'), # unicode for ^2
-        ('axon_perimeter (um)', 'f4'),
-        ('myelin_area (um\u00b2)', 'f4'),
-        ('axon_diam (um)', 'f4'),
-        ('myelin_thickness (um)', 'f4'),
-        ('axonmyelin_area (um\u00b2)', 'f4'),
-        ('axonmyelin_perimeter (um)', 'f4'),
-        ('solidity', 'f4'),
-        ('eccentricity', 'f4'),
-        ('orientation', 'f4'),
-    ]
-)
+
+class Morphometrics_Column_Name:
+    """
+    A Morphometrics_Column_Name indicates the key_name and display_name of a morphometrics dataframe column
+    :param key_name: The column name used in the dataframe variable
+    :type key_name: str
+    :param display_name: (optional) The column name that will be displayed in the csv/excel file. This is useful if you
+    want to specify units to the user
+    :type display_name: str
+    """
+    def __init__(self, key_name, display_name=None):
+        self.key_name = key_name
+        self.display_name = display_name
+
```

Comment:
I placed this class definition in params.py as this is where it is used, but I'm wondering if I should move it somewhere else. And if so, where?